### DISCUSSION TITLE
Prisma inside docker takes almost 300mb

### DISCUSSION THREAD

**Discussion by vertti** [prisma/prisma#11572] [Q&A] [answered]
I was digging through my docker container that contains a simple app that uses Prisma.

Under `node_modules` you can see Prisma taking space like this:
<img width="831" alt="Screenshot 2022-02-02 at 17 06 03" src="https://user-images.githubusercontent.com/557751/152181017-8ed72d1f-98c5-414d-9720-9776de1a2d9d.png">

total of 127MB. But slightly surprisingly, another 114 MB of Prisma stuff is found under `.cache`:
<img width="809" alt="Screenshot 2022-02-02 at 17 06 45" src="https://user-images.githubusercontent.com/557751/152181160-0ad3bff9-b0c8-443c-ac83-e7287a62586a.png">

Are the `.cache` files necessary? Can I somehow avoid pulling those in? The total added size of Prisma is quite a lot!

**Comment by nemanjam** [comment]
I am interested in this too.

**Comment by nikolasburk** [comment, marked as answer]
Thanks a lot for raising this @vertti! I've checked with our engineers and it seems like we can probably improve the situation, so I created an issue here: https://github.com/prisma/prisma/issues/11577

You can also safely delete anything in the `.cache` directory if you want to save some space already 🙏

**Comment by nemanjam** [comment]
I asked similar question in this topic: https://github.com/prisma/prisma/discussions/11178

You can clear the yarn cache in Dockerfile but it still will be 200MB. If you plan to keep `"prisma": "^3.8.0"` as dev dependency you will need separate container for migration and seed, but to my calculation that wont save much space, although I tried it out only with yarn on the host and not in Docker, I didn't spend too much time on this. You can see my first post for details.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
